content/10.diagnostics.md

```diff
@@ -182,61 +211,68 @@ Circulating antibody titers to other coronaviruses have been reported to decline
+
+Concerns have also been raised that immunity may wane over time.
+Several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
+Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection.
+Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481].
+<!--To Do: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
 
 #### Current Approaches
 
+<!-- To Do: Update for 2021-->
 Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions [@url:https://www.fda.gov/media/136625/download].
 Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma (from a blood sample) of patients suspected to have developed the SARS-CoV-2 infection [@url:https://www.fda.gov/media/136625/download].
 Such tests illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
 Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
-The test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
+The Cellex test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
 In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
-With this particular assay, results can be read within 15-20 minutes [@url:https://www.fda.gov/media/136625/download].
+With this particular assay, results can be read within 15 to 20 minutes [@url:https://www.fda.gov/media/136625/download].
 Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai, proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
```

Comment:
```suggestion
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai, proposed an ELISA test that detects IgG and IgM that react against the RBD of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
```

Reply:
Maybe just "S" here, although it looked weird with I wrote it.

Reply:
I'll go through and clean all the abbreviations up at some point - I usually do that on the day we submit because they are usually changing up until pretty close to submission!!